domain/src/main/java/com/kimdoori/githubapp/domain/usecase/GitHubRepoUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.kimdoori.githubapp.domain.usecase
+
+import com.kimdoori.githubapp.data.GitHubRepoRepository
+import io.reactivex.rxjava3.core.Single
+import com.kimdoori.githubapp.domain.model.GitHubRepoModel
+
+class GitHubRepoUseCase(private val gitHubRepoRepository: GitHubRepoRepository) {
```

Comment:
```suggestion
class GitHubRepoUseCase @Inject constructor(private val gitHubRepoRepository: GitHubRepoRepository) {
```

`@Inject` 어노테이션이 있어야만 dagger 에서 injection 이 됩니다 😄 
DataSource, Repository 에도 적용시켜주세요 🙏

Reply:
앗... 체크 감사합니다!!

Reply:
https://github.com/kimdoori/GitHubApp/pull/16/commits/440946ebabe54bb9ffc860dce09ef50fcf919899 적용했습니다!